vlib/v/TEMPLATES.md

```diff
@@ -157,3 +157,9 @@ where you can insert your src
 
 All variables, which are declared before the $tmpl can be used through the `@{my_var}` syntax.
 It's also possible to use properties of structs here like `@{my_struct.prop}`.
+
+# Escaping
+
+The `@` sign is used to begin a template directive. 
+To be able to use the `@` sign as a normal character inside a 
+template use `$$`.
```

Comment:
Hm, can you show a more complete example?

Reply:
Yes. Here are an example. If you place this in a template:

<link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" crossorigin=""/>

it will render as:

<link rel="stylesheet" href="https://unpkg.com/leaflet$1.9.4/dist/leaflet.css" crossorigin=""/>

because of vlib/v/parser/tmpl.v line 80-82.

If you need to use @ as a normal @ char in a template you must use $$ instead. That is, if you place this in a template:

<link rel="stylesheet" href="https://unpkg.com/leaflet$$1.9.4/dist/leaflet.css" crossorigin=""/>

if renders as:

<link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" crossorigin=""/>

Reply:
Did you try `@@` to get a single `@` in the output?

Reply:
I have now, @@ works as well. I have updated the description to indicate that @@ should be used.

Reply:
Thank you.

Reply:
It is strange to me that `$$` worked and turned into `@`... :man_shrugging: